src/Concordium/Client/Output.hs

```diff
@@ -912,14 +911,16 @@ showEvent verbose ciM = \case
     showLoggedEvents :: [Wasm.ContractEvent] -> String
     showLoggedEvents [] = "No contract events were emitted."
     showLoggedEvents evs =
-        [i|#{length evs} contract events were emitted|]
+        [i|#{length evs} contract #{if length evs > 1 then "events were" else ("event was" :: String)} emitted|]
             <> ( if isNothing eventSchemaM
                     then [i| but no event schema was provided nor found in the contract module. |]
-                    else [i|, of which #{length $ filter isLeft $ map showContractEvent evs} were succesfully parsed. |]
+                    else [i|, of which #{noOfParsedEvents} #{if noOfParsedEvents > 1 then "were" else ("was" :: String)} successfully parsed. |]
```

Comment:
Suggestion: instead of `> 1`, use `/= 1` so that the `0` case takes the plural verb.

Reply:
The 0 case is handled above:

```hs
  showLoggedEvents [] = "No contract events were emitted."
```

But I still think `/= 1` is better.